src/api/controllers/articleController.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
I disabled this eslint rule cause we had no specific rule for it in the eslint file, and if I didn't wanna disable the whole file in the eslintignore file cause nobody knows what the future holds... so your Ideas are welcome

Reply:
@MCFrank16 have you implemented @nkpremices' feedback above?

Reply:
I was waiting for your go ahead, so now am gon change it